rpcs3/util/logs.cpp

```diff
@@ -412,7 +412,7 @@ logs::file_writer::file_writer(const std::string& name, u64 max_size)
 	{
 		thread_ctrl::set_native_priority(-1);
 
-		while (true)
+		while (m_out != umax)
```

Comment:
I don't think this change is correct.

But I found out that this causes file_writer::flush to endlessly return false:
`if (end > st) // end == st == 363895`

363895 is obviously way smaller than umax

Reply:
this change changes nothing actually.
need to check if the destructor is called at all or m_out signaling gets discarded.